todo_app/Tests/test_application.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pytest
+import requests
+from todo_app import app
+from dotenv import load_dotenv, find_dotenv
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
+
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+
+# Stub replacement for requests.get(url)
+def stub(method,url, params):
+    test_board_id = os.environ.get('TRELLO_BOARDID')
+    fake_response_data = None
+    if method == 'GET' and url == f'https://api.trello.com/1/boards/{test_board_id}/cards':
+        fake_response_data = [{'id': '456', 'name': 'Test card', 'idList': '61bb961c223edf0a94b3973f'}]
```

Comment:
```suggestion
        fake_response_data = [{'id': '456', 'name': 'Test card', 'idList': 'trello-listid1'}]
```

Could use one of your ids in .env.test rather than a real one. Then your test could assert the card is displayed as "to do". 

No need to bother though!